packages/table/src/quadrants/_quadrants.scss

```diff
@@ -51,9 +56,22 @@ $table-quadrant-scroll-container-overflow: 20px;
 }
 
 .bp-table-quadrant-main {
-  right: 0;
-  bottom: 0;
+  // use `relative` to ensure the table can size itself according to the MAIN quadrant's contents alone.
+  // other quadrants should remain `absolute` to ensure they're taken out of the document flow.
+  position: relative;
+  top: auto;
+  left: auto;
   z-index: $table-quadrant-z-index-main;
+  // the MAIN quadrant should fill the whole table so that ghost cells will be visible if enabled
+  width: 100%;
+  height: 100%;
```

Comment:
`height: 100%` can be tricky sometimes. just want to make sure this works correctly.

```html
<div>
    <div style="height: 100%">everything is cool</div>
    <div>what now?</div>
</div>
```

 this should be okay since we control the siblings of this div, but just something to keep an eye on.